src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -7,20 +7,42 @@
 public class Shooter extends SubsystemBase
 {
   private Victor motor;
+  private double shooterSpeed;
```

Comment:
Just a suggestion but you can just call it `speed`.  Obviously it is the shooter speed cause it's in the Shooter subsystem.